src/common/constants.ts

```diff
@@ -26,11 +26,13 @@ export class Constants {
     public static LANGUAGE_NODE = "Node.js Module";
     public static LANGUAGE_PYTHON = "Python Module";
     public static CSHARP_FUNCTION = "Azure Functions - C#";
+    public static ACR_MODULE = "Existing Module on Azure Container Registry";

```

Comment:
M lower case?

Reply:
I use title case to align with other options.